tlslite/keyexchange.py

```diff
@@ -318,24 +318,23 @@ def makeServerKeyExchange(self, sigHash=None):
         self.signServerKeyExchange(ske, sigHash)
         return ske
 
-# The ECDHE_RSA part comes from the IETF names of ciphersuites, so we want to
-# keep it
-#pylint: disable = invalid-name
-class ECDHE_RSAKeyExchange(KeyExchange):
-    """Helper class for conducting ECDHE key exchange"""
+class AECDHKeyExchange(KeyExchange):
+    """
+    Handling of anonymous Eliptic curve Diffie-Hellman Key exchange
 
-    def __init__(self, cipherSuite, clientHello, serverHello, privateKey,
-                 acceptedCurves):
-        super(ECDHE_RSAKeyExchange, self).__init__(cipherSuite, clientHello,
-                                                   serverHello, privateKey)
+    ECDHE without signing serverKeyExchange useful for anonymous ECDH
+    """
+    def __init__(self, cipherSuite, clientHello, serverHello, acceptedCurves):
+        super(AECDHKeyExchange, self).__init__(cipherSuite, clientHello,
+                                               serverHello)
 #pylint: enable = invalid-name
```

Comment:
since code is not disabling the invalid-name check, it doesn't need to re-enable it